### PR TITLE
Add ReadMe.md

### PR DESCRIPTION
**Pull request by ChrisOwen101**
This closes ticket #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# CrazyCounter
\ No newline at end of file
+# CrazyCounter
+
+This is a simple counter app that demonstrates the use of the `useState` hook in React.
+
+## Installation
+
```
Comment (Kurt812) [inline README.md:6]: Are there any requirements?

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
-# CrazyCounter
\ No newline at end of file
+# CrazyCounter
```
Comment (gem09lo) [inline README.md:1]: good

File: README.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+
+This is a simple counter app that demonstrates the use of the `useState` hook in React.
+
+## Installation
+
+1. Clone the repository
```
Comment (GabrielNsiah) [inline README.md:7]: Maybe add a link to the repository?

File: README.md [hunk 4]
```diff
@@ -1 +1,16 @@
...
+2. Run `npm install`
+3. Run `npm start`
+4. Visit `http://localhost:3000/`
+5. Enjoy!
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details.
```
Comment (bentrzcinski) [inline README.md:15]: Congrats on the job at MIT!

File: README.md [hunk 5]
```diff
@@ -1 +1,16 @@
-# CrazyCounter
\ No newline at end of file
+# CrazyCounter
+
+This is a simple counter app that demonstrates the use of the `useState` hook in React.
+
+## Installation
+
+1. Clone the repository
+2. Run `npm install`
```
Comment (jk-george) [inline README.md:8]: what's npm

File: README.md (hunk 4 above)
Comment (jiuliangut) [inline README.md:15]: [insert useful comment here]

File: README.md [hunk 6]
```diff
@@ -1 +1,16 @@
...
+3. Run `npm start`
+4. Visit `http://localhost:3000/`
+5. Enjoy!
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details.
+
```
Comment (emily-curtis) [inline README.md:16]: more detail would be good?

File: README.md [hunk 7]
```diff
@@ -1 +1,16 @@
-# CrazyCounter
\ No newline at end of file
+# CrazyCounter
+
+This is a simple counter app that demonstrates the use of the `useState` hook in React.
+
```
Comment (SurinaCS) [inline README.md:4]: are there any prerequisites?

File: README.md [hunk 8]
```diff
@@ -1 +1,16 @@
...
+## Installation
+
+1. Clone the repository
+2. Run `npm install`
+3. Run `npm start`
+4. Visit `http://localhost:3000/`
+5. Enjoy!
+
```
Comment (Jakub-Poskrop) [inline README.md:12]: Are there any environmental variables?

File: README.md (hunk 6 above)
Comment (anitamegarry) [inline README.md:16]: Looks good to me

**Comment by Fahi28** [comment]
nice

File: README.md [hunk 9]
```diff
@@ -1 +1,16 @@
-# CrazyCounter
\ No newline at end of file
+# CrazyCounter
+
+This is a simple counter app that demonstrates the use of the `useState` hook in React.
```
Comment (qrafiq1) [inline README.md:3]: Nice full stop.

File: README.md [hunk 10]
```diff
@@ -1 +1,16 @@
...
+
+## Installation
```
Comment (ebradley12) [inline README.md:5]: Great set up guide!

File: README.md [hunk 11]
```diff
@@ -1 +1,16 @@
...
+
+1. Clone the repository
+2. Run `npm install`
+3. Run `npm start`
```
Comment (cqmeronn) [inline README.md:9]: Need to install anything for npm?

File: README.md [hunk 12]
```diff
@@ -1 +1,16 @@
...
+4. Visit `http://localhost:3000/`
+5. Enjoy!
```
Comment (keoghrmj) [inline README.md:11]: Lovely stuff

Comment (S1mpySloth) [inline README.md:11]: Any pre-requirements?

File: README.md (hunk 2 above)
Comment (Fahi28) [inline README.md:1]: nice

**Review by gem09lo — APPROVED** [review]

**Review by Kurt812 — APPROVED** [review]
Outstanding!!

**Review by keoghrmj — APPROVED** [review]
Amazing. Couldn't have written it better myself

**Review by qrafiq1 — APPROVED** [review]
Nice

**Review by lukieh2014 — COMMENTED** [review]
pretty good stuff, keep it up

**Review by Jakub-Poskrop — APPROVED** [review]
Good work

**Review by jk-george — APPROVED** [review]
approved

**Review by RidwanHamid501 — APPROVED** [review]
Definitely add this. Really good!

**Review by Fahi28 — APPROVED** [review]
nice work

**Review by emily-curtis — APPROVED** [review]

**Review by S1mpySloth — APPROVED** [review]
Nice work

**Review by anitamegarry — APPROVED** [review]
Great

**Review by jiuliangut — APPROVED** [review]

**Review by GabrielNsiah — APPROVED** [review]

**Review by lukieh2014 — APPROVED** [review]
nice stuff

**Review by gem09lo — APPROVED** [review]
Nice work

**Review by ebradley12 — APPROVED** [review]
great!

**Review by cqmeronn — APPROVED** [review]

**Pull request merged**